rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -27,13 +27,13 @@ def coco(*inputs, file_root, annotations_file='', bytes_per_sample_hint=0, dump_
          meta_files_path='', num_shards=1, pad_last_batch=False, prefetch_queue_depth=1, preserve=False,
          random_shuffle=False, ratio=False, read_ahead=False, save_img_ids=False, seed=-1, shard_id=0,
          shuffle_after_epoch=False, size_threshold=0.1, skip_cached_images=False, skip_empty=False,
-         stick_to_shard=False, tensor_init_bytes=1048576):
+         stick_to_shard=False, tensor_init_bytes=1048576, is_box_encoder=False, is_box_iou_matcher=False):
 
     Pipeline._current_pipeline._reader = "COCOReader"
     #Output
     labels = []
     bboxes = []
-    kwargs_pybind = {"source_path": annotations_file, "is_output":True}
+    kwargs_pybind = {"source_path": annotations_file, "is_output":True, "is_box_encoder":is_box_encoder, "is_box_iou_matcher":is_box_iou_matcher }
```

Comment:
is this parameters similar to DALI?

Reply:
No Rajy, this has been introduced in rocAL